### PR TITLE
Feature/sc 539

### PR DESCRIPTION
**Pull request by DanCh11**

### CODE REVIEW

File: sem_covid/entrypoints/etl_dags/crawl_pipeline.py [hunk 1]
```diff
@@ -76,7 +76,7 @@ def __init__(self, store_registry: StoreRegistryABC, file_name: str, bucket_name
         self.scrapy_crawler = scrapy_crawler
 
     def get_steps(self) -> list:
-        return [self.extract, self.transform_structure,
+        return [self.extract, self.transform_content,
```
Comment (costezki) [inline sem_covid/entrypoints/etl_dags/crawl_pipeline.py:79]: we shall call the transform structure, even if it is empty. 
All ETL steps shall be executed.

Comment (costezki) [reply sem_covid/entrypoints/etl_dags/crawl_pipeline.py:79]: better inherit this method from the Base.

File: sem_covid/services/crawlers/scrapy_crawlers/spiders/eu_timeline_spider.py [hunk 2]
```diff
@@ -14,7 +14,7 @@ class EUTimelineSpider(scrapy.Spider):
     name = 'eu-timeline'
     base_url = 'https://ec.europa.eu'
     url = 'https://ec.europa.eu/info/live-work-travel-eu/coronavirus-response/timeline-eu-action_en'
-    presscorner_base_url = 'https://ec.europa.eu/commission/presscorner/detail'
+    presscorner_base_url = 'https://ec.europa.eu'
```
Comment (costezki) [inline sem_covid/services/crawlers/scrapy_crawlers/spiders/eu_timeline_spider.py:17]: I still do not understand why we need this. Can we delete this line? 
If not, then why?

File: sem_covid/services/crawlers/scrapy_crawlers/spiders/eu_timeline_spider.py [hunk 3]
```diff
@@ -88,9 +88,12 @@ def parse_presscorner_page(self, response):
...
         metadata = response.xpath('//span[contains(@class, "ecl-meta__item")]//text()')
-        item['detail_type'] = metadata[0].get()
-        item['detail_date'] = metadata[1].get()
-        item['detail_location'] = metadata[2].get()
+        if metadata:
+            item['detail_type'] = metadata[0].get()
+            item['detail_date'] = metadata[1].get()
+            item['detail_location'] = metadata[2].get()
+        else:
+            print("No detail type, date and/or location found on this page. . .")
```
Comment (costezki) [inline sem_covid/services/crawlers/scrapy_crawlers/spiders/eu_timeline_spider.py:96]: never print, always use `logger.info`, `logger.warning`, `logger.error`

File: tests/unit/dags/test_crawl_eu_action_timeline_dag.py [hunk 4]
```diff
@@ -13,22 +13,22 @@ def test_crawl_eu_action_timeline_has_three_tasks_and_order(airflow_dag_bag):
     assert dag is not None
     tasks = dag.tasks
     task_ids = list(map(lambda task: task.task_id, tasks))
-    assert {'extract', 'transform_structure', 'load'}.issubset(set(task_ids))
+    assert {'extract', 'transform_content', 'load'}.issubset(set(task_ids))
```
Comment (costezki) [inline tests/unit/dags/test_crawl_eu_action_timeline_dag.py:16]: one more method is missing

**Review by costezki — CHANGES_REQUESTED** [review]
please address the comments

File: sem_covid/services/crawlers/scrapy_crawlers/spiders/eu_timeline_spider.py (hunk 2 above)
Comment (DanCh11) [reply sem_covid/services/crawlers/scrapy_crawlers/spiders/eu_timeline_spider.py:17]: which line? base_url?

File: tests/unit/dags/test_crawl_eu_action_timeline_dag.py (hunk 4 above)
Comment (DanCh11) [reply tests/unit/dags/test_crawl_eu_action_timeline_dag.py:16]: but the test will fail

**Pull request merged**